### ISSUE TITLE
Dependency on Ruuid

### ISSUE THREAD

**Issue by louisaslett** [WLOGSolutions/r-logging#16]
Ruuid doesn't belong to the current release or devel version of Bioconductor anymore so I am getting errors when trying to install it on R 3.0.1.  Should this dependency be removed since BioC seem to be deprecating it?

Louis

**Comment by louisaslett** [comment]
Sorry I should have mentioned that I'm talking about the Sentry logging handler here! :)

Louis

**Comment by mfrasca** [comment]
soon after I decided using it it became deprecated, but I didn't get the time to find out how to do without it.
if you have a patch that solves the issue, I will be glad to accept it. 
(fork the project, correct, push, file a pull-request).
I am not currently working at nor even with this library, I'm just maintaining it if bugs arise and users have patches.

**Comment by mfrasca** [comment]
hi. it this still an issue?

**Comment by anosh7133** [comment]
Which problem are you talking about?

**Comment by louisaslett** [comment]
> hi. it this still an issue?

I'm not entirely sure, this was a long time ago and I've moved on from the work I needed it for. However, a quick search of my own past code and this repo shows the file I was talking about was this:

https://github.com/WLOGSolutions/r-logging/blob/f8ca2471cdb6d8c762439d9cd5a671891e621ada/handlers/pkg/R/sentry.R#L32-L40

That dependency on `Ruuid` cannot be fulfilled and the code to install from Bioconductor does not work. In that sense, there is still a problem in code that exists at the current head of this repository.

**Comment by mfrasca** [comment]
as far as I can see, the only use of requiring Ruuid is for calling getuuid.
replacing with `uuid::UUIDgenerate` should work.

**Comment by louisaslett** [comment]
Agreed, that looks a promising possible fix. 👍 However, in honesty I don't need this functionality for any of my current work so don't really have the capacity to test it.
